### PR TITLE
About Me Page Part

### PR DESCRIPTION
**Pull request by kv154chigo**
First pull request for the about me part of the project.

### CODE REVIEW

**Review by SeanCampbell — COMMENTED** [review]
Could you exclude the files that aren't directly related to the About Me Page project?

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+src/main/java/com/google/codeu/servlets/AboutMeServlet.java
```
Comment (SeanCampbell) [inline .gitignore]: These should be the files to exclude from tracking, so target/

File: src/main/java/com/google/codeu/data/Datastore.java [hunk 2]
```diff
@@ -27,6 +27,8 @@
 import java.util.List;
 import java.util.UUID;
 
+
```
Comment (SeanCampbell) [inline src/main/java/com/google/codeu/data/Datastore.java]: Super nitpicky comment :) Better to leave out these extra newlines to focus on the changes to functionality.

**Review by SeanCampbell — CHANGES_REQUESTED** [review]
Mostly looks good to me! I think it should be good once the .gitignore file is updated, but I'll just take another look then to make sure I didn't miss anything.

File: .gitignore (hunk 1 above)
Comment (kv154chigo) [reply .gitignore]: So instead of the files I currently have in .gitignore, it should just be "target/" to exclude all of the files from that folder?

**Review by SeanCampbell — APPROVED** [review]
Nice work! This looks good to me; feel free to merge this PR.

Don't forget to run "git pull" to get the newest updates from the master branch before merging in the PR. You might get conflicts in the .gitignore file, in which case, just add on whatever you have that isn't already in the master's .gitignore.

**Comment by kv154chigo** [comment]
I am trying to merge the PR but its stating that "This branch has conflicts that must be resolved"

**Pull request merged**